### PR TITLE
Fix <Switch /> small margin

### PR DESCRIPTION
**Pull request by bryanberger**
This should be 13px. Size of the circle (12px) + spacing to match the left size(1px).
https://github.com/ant-design/ant-design/blob/master/components/switch/style/index.less#L110

**Issue using current value of -12.5px:**
screen shot 2018-03-09 at 2 26 22 pm

**Issue solved:**
screen shot 2018-03-09 at 2 26 17 pm

* [x] branch master
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by bryanberger** [comment]
Hi, any thoughts on this?

**Review by yutingzhao1991 — APPROVED** [review]

File: components/switch/style/index.less [hunk 1]
```diff
@@ -107,7 +107,7 @@
     &:before,
     &:after {
       left: 100%;
-      margin-left: -12.5px;
+      margin-left: -(@switch-sm-height - 3px);
```
Comment (valleykid) [inline components/switch/style/index.less]: Sorry to have a little doubt: Why not define a new variable but use the `switch-sm-height`(Inconsistent with single responsibility)?
ps: `.5px` may be related to browser compatibility. (cc @afc163 )

Comment (bryanberger) [reply components/switch/style/index.less]: Ah, sure thing. I noticed currently that variable was being used with multiple responsibility.

**Comment by bryanberger** [comment]
@yutingzhao1991 @valleykid I added a new var for single responsibility. Not sure what CodeCov is failing for, thoughts?

**Comment by yesmeck** [comment]
Thanks @bryanberger

**Pull request merged**